src/document/HolonomicPathStore.ts

```diff
@@ -20,9 +20,14 @@ import {
   WaypointID,
   WaypointScope,
 } from "./ConstraintStore";
-import { SavedWaypointId } from "./previousSpecs/v0_1";
+import { SavedWaypointId } from "./previousSpecs/v0_1_2";
```

Comment:
This should actually be forward-exported from DocumentSpecTypes, but that's outside the scope of this PR